Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:10
+
+WORKDIR /app
+
+COPY . /app
+
+RUN yarn install
+
+CMD node index.js
+
+EXPOSE 8080
```

Comment:
let's use another port, e.g. `3000`, because `8080` port used for browser apps mostly